src/ci/docker/scripts/x86_64-gnu-llvm1.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -ex
+
+./add_dummy_commit.sh
+
+../x.py --stage 2 test \
+  --skip tests \
+  --skip coverage-map \
+  --skip coverage-run \
+  --skip library \
+  --skip tidyselftest
```

Comment:
These are duplicated from the CI [sets](https://github.com/MarcoIeni/rust/blob/c2f45505f53749548ea89e1a46b7f8c036f23b1f/src/ci/github-actions/jobs.yml#L62).

One way to de-duplicate them might be to move the CI anchors to scripts.
We could do this in another PR. What do you think?

Reply:
I'm not sure if the scripts could encompass everything, e.g. the Windows runners don't use bash scripts within Docker, some runners still use a Makefile.. Let's leave it like this for now.